Src/AutoFixture/PropertyBasedInstanceGenerator.cs

```diff
@@ -1,63 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Reflection;
-
-namespace Ploeh.AutoFixture
-{
-    /// <summary>
-    /// Generates instances based on <see cref="PropertyInfo"/> instances.
-    /// </summary>
-    public class PropertyBasedInstanceGenerator : InstanceGeneratorNode

```

Comment:
This orphaned file seems to be the result of a botched file move in d9bf1053d04742d588fcf59c76035bb4c6241325. Instead of moving the file, it looks like I copied it to the new destination.

I'll remove it from *master* shortly.